DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/AlignParam.h

```diff
@@ -52,7 +52,7 @@ class AlignParam
   double getZ() const { return mZ; }
 
   /// apply object to geoemetry
-  bool applyToGeometry() const;
+  bool applyToGeometry(bool usePW = false, bool addSensorPW = false, bool addMetalPW = false, bool addChipPW = false) const;
```

Comment:
In principle this function has nothing to do with ITS. Yet, we pass several ITS specific arguments to it.
Could we think of a cleaner structure? Something like passing a AlignGeomOptions struct (which is then extensible)? The struct could contain ITS specific members.